index.html

```diff
@@ -40,12 +48,12 @@ <h2 class="activity-title" id="new-activity">New Activity</h2>
           <input class="user-input" id="user-seconds" type="number" name="seconds"/></label>
         </div>
         <button id="start-activity">START ACTIVITY</button>
-      </form>
+      </section>
+      <aside id="activity-log">
```

Comment:
Maybe we can line the aside class attribute to be something inline with "big-box"